generator/internal/dart/dart.go

```diff
@@ -174,9 +185,23 @@ func enumValueName(e *api.EnumValue) string {
 	return strcase.ToLowerCamel(e.Name)
 }
 
-func httpPathFmt(_ *api.PathInfo) string {
+func httpPathFmt(pathInfo *api.PathInfo) string {
 	fmt := ""
-	// TODO(#1034): Determine the correct format for Dart.
+	for _, segment := range pathInfo.PathTemplate {
+		if segment.Literal != nil {
```

Comment:
I think the golang folks prefer `switch` and `case` over `if / else if`.   There is also this:

https://pkg.go.dev/strings#Builder

that could be handy.

Reply:
Thanks for the ptr to Builder! And, I'd seen that use for switch statements but hadn't started using it yet - I moved this code over to that pattern.